x-pack/plugins/security_solution/server/endpoint/routes/metadata/metadata.test.ts

```diff
@@ -113,552 +119,616 @@ describe('test endpoint route', () => {
     });
   });
 
-  describe('with .metrics-endpoint.metadata_united_default index', () => {
-    beforeEach(() => {
-      endpointAppContextService = new EndpointAppContextService();
-      mockPackageService = createMockPackageService();
-      mockPackageService.getInstallation.mockReturnValue(
-        Promise.resolve({
-          installed_kibana: [],
-          package_assets: [],
-          es_index_patterns: {},
-          name: '',
-          version: '',
-          install_status: 'installed',
-          install_version: '',
-          install_started_at: '',
-          install_source: 'registry',
-          installed_es: [
-            {
-              id: 'logs-endpoint.events.security',
-              type: ElasticsearchAssetType.indexTemplate,
-            },
-            {
-              id: `${metadataTransformPrefix}-0.16.0-dev.0`,
-              type: ElasticsearchAssetType.transform,
-            },
-          ],
-          keep_policies_up_to_date: false,
-        })
-      );
-      endpointAppContextService.setup(createMockEndpointAppContextServiceSetupContract());
-      endpointAppContextService.start({ ...startContract, packageService: mockPackageService });
-      mockAgentService = startContract.agentService!;
-      mockAgentPolicyService = startContract.agentPolicyService!;
+  describe('POST list endpoints route', () => {
```

Comment:
most of this is actually untouched, I just moved/nested some stuff. also duplicated POST API tests then modified them for the new GET API.